File: clients/android/app/src/main/java/com/example/mooc/Entities/OfferSubject.java
package com.example.mooc.Entities;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class OfferSubject implements Serializable {

    @SerializedName("id")
    private int id;

    @SerializedName("price")
    private float price;

    @SerializedName("nbSchoolSubject")
    private int nb_school_subject;

    public int getId() {
        return id;
    }

    public float getPrice() {
        return price;
    }

    public int getNb_school_subject() {
        return nb_school_subject;
    }
}

File: server/src/Entity/OfferSubject.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * SubjectOffer
 *
 * @ORM\Table(name="offer_subject")
 * @ORM\Entity
 */
class SubjectOffer
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="bigint", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var float
     *
     * @ORM\Column(name="price", type="float", precision=10, scale=0, nullable=false)
     */
    private $price;

    /**
     * @var int
     *
     * @ORM\Column(name="nb_school_subject", type="integer", nullable=false)
     */
    private $nbSchoolSubject;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getPrice(): ?float
    {
        return $this->price;
    }

    public function setPrice(float $price): self
    {
        $this->price = $price;

        return $this;
    }

    public function getNbSchoolSubject()
    {
        return $this->nbSchoolSubject;
    }

    public function setNbSchoolSubject($nbSchoolSubject): self
    {
        $this->nbSchoolSubject = $nbSchoolSubject;

        return $this;
    }

}
